res/opensysroot/opensysroot/__init__.py

```diff
@@ -52,6 +52,15 @@ def main():
         assert args.arch is not Arch.CORTEXA9
         db.add_package("gcc")
         db.add_package("g++")
+
+        # Packages for GLFW
+        db.add_package("libx11-dev")
+        db.add_package("libxcursor-dev")
+        db.add_package("libxrandr-dev")
+        db.add_package("libxinerama-dev")
+        db.add_package("libxi-dev")
+        db.add_package("mesa-common-dev")
+
         if args.minimal_toolchain:
```

Comment:
Would it make sense to put these extra packages under the “minimal toolchain” set, or does that flag do something different?

Reply:
The minimal toolchain flag is an artifact of when I was manually rebuilding the GCC libraries. Now that everything is pulled from the repositories, this no longer applies. The minimal toolchain notion could be reworked to optionally have XOrg, but this would involve reworking the targets to support flavors.

I don't imagine much people use the toolchains outside the roboRIO toolchain. So, I think we can be a bit more flexible with how big the final package can get.